.aicoe-ci.yaml

```diff
@@ -2,7 +2,7 @@ check:
   - thoth-build
   - thoth-precommit
 build:
-  base-image: quay.io/thoth-station/s2i-thoth-ubi8-py38:latest
+  base-image: quay.io/thoth-station/s2i-thoth-ubi8-py38::v0.20.1
```

Comment:
there is an extra `:` in the tag, please fix it.

Reply:
thanks, fixed